build.gradle

```diff
@@ -131,6 +130,10 @@ dependencies {
   )
 }
 
+test {
+  environment("APPINSIGHTS_INSTRUMENTATIONKEY", "test-key")
```

Comment:
Do we need to set instrumentation key for functional tests?

Reply:
good question. not sure what's set as environment vars since all directory of functional tests are not for local execution. even integration test uses non-mocked third party components

on the other hand, we set in other projects too. tests are not against some running instance but the actual code within source. yep, answered my own wonders :D